irohad/main/impl/ordering_init.hpp

```diff
@@ -57,11 +60,11 @@ namespace iroha {
        * @param loop - handler of async events
        */
```

Comment:
Please, update this comment and check other comments in this file. I see `loop` parameter in comment below, which does not exists